src/containers/SemesterReserveContainer.jsx

```diff
@@ -1,22 +1,50 @@
-import { connect } from 'react-redux';
+import { connect as connectRedux } from 'react-redux';
+import { DropTarget } from 'react-dnd';
 import SemesterReserve from '../components/SemesterReserve';
 
 const filterCourse = (plannedCourses) => {
+  const compare = (a, b) => {
+    if (a.registrarId.toUpperCase() < b.registrarId.toUpperCase()) {
+      return -1;
+    }
+    return 1;
+  };
   const filteredPlanned = plannedCourses.filter(course =>
     course.semester === undefined,
-  );
+  ).sort(compare);
   return filteredPlanned;
 };
 
-const mapStateToProps = (state, ownProps) => (
+// How the target reacts to drag and drop events
+const semesterTarget = {
+  drop(props, monitor) {
+    // return drop result available to the drag sourc
```

Comment:
Not the way the state is structured now as far as I can tell... We were combining reducers in the wrong way. I'm working on improving that on another card. I'll fix the 'e'!